### PR TITLE
[sai-redis] Add policer and Trap Group serialization

### PR DESCRIPTION
**Pull request by hrachya-mellanox**
* Added policer and trap group serialization for COPP.
Signed-off-by: hrachya@mellanox.com

### CODE REVIEW

File: common/saiserialize.cpp [hunk 1]
```diff
@@ -100,12 +100,34 @@ sai_serialization_map_t sai_get_serialization_map()
...
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_CIR] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_PBS] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_PIR] = SAI_SERIALIZATION_TYPE_UINT64;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_GREEN_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_YELLOW_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_RED_PACKET_ACTION] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_POLICER][SAI_POLICER_ATTR_ENABLE_COUNTER_LIST] = SAI_SERIALIZATION_TYPE_INT32_LIST;
+
+    map[SAI_OBJECT_TYPE_TRAP_GROUP][SAI_HOSTIF_TRAP_GROUP_ATTR_ADMIN_STATE] = SAI_SERIALIZATION_TYPE_BOOL;
+    //map[SAI_OBJECT_TYPE_TRAP_GROUP][SAI_HOSTIF_TRAP_GROUP_ATTR_QUEUE] = SAI_SERIALIZATION_TYPE_UINT32;
```
Comment (stcheng) [inline common/saiserialize.cpp]: why this line is commented?

Comment (hrachya-mellanox) [inline common/saiserialize.cpp]: I've tested code with SAI_HOSTIF_TRAP_GROUP_ATTR_PRIO which should be removed soon and replaced with SAI_HOSTIF_TRAP_GROUP_ATTR_QUEUE new attribute.
It's not present in the current saitypes.h installed on the switch.
Forgot to delete the commented line.

Comment (lguohan) [inline common/saiserialize.cpp]: can you delete it?

File: common/saiserialize.cpp [hunk 2]
```diff
@@ -100,12 +100,34 @@ sai_serialization_map_t sai_get_serialization_map()
...
     map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_PRIORITY] = SAI_SERIALIZATION_TYPE_UINT32;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_CHANNEL] = SAI_SERIALIZATION_TYPE_INT32;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_FD] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_PORT_LIST] = SAI_SERIALIZATION_TYPE_OBJECT_LIST;
+    map[SAI_OBJECT_TYPE_TRAP][SAI_HOSTIF_TRAP_ATTR_TRAP_GROUP] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
 
     map[SAI_OBJECT_TYPE_LAG_MEMBER][SAI_LAG_MEMBER_ATTR_LAG_ID] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
     map[SAI_OBJECT_TYPE_LAG_MEMBER][SAI_LAG_MEMBER_ATTR_PORT_ID] = SAI_SERIALIZATION_TYPE_OBJECT_ID;
 
+
```
Comment (stcheng) [inline common/saiserialize.cpp]: extra empty line

**Pull request closed without merging**